extensions/apicurio-registry-avro/deployment/pom.xml

```diff
@@ -23,6 +23,12 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core-deployment</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-kubernetes-service-binding-deployment</artifactId>
```

Comment:
@geoand is it how it's intended to work? 
Typically, the kafka client deployment module does not have a dependency on the service binding deployment.

Reply:
@cescoffier, you are right, this is **not** how it's intended to work. The kafka client and others (like the mongo client - `quarkus-mongodb-client`) have an optional dependency in the runtime module